api.planx.uk/modules/analytics/metabase/service.ts

```diff
@@ -9,34 +9,38 @@ import { getTeamSlug } from "./getTeamSlug.js";
  * @returns The dashboard name (the Metabase API performs GETs with the dashboard ID, so we have to have that locally already--no need to return it here)
  */
 export async function createNewDashboardLink({
+  analyticsLink,
+  status,
   flowId,
   teamId,
   serviceSlug,
 }: CreateNewDashboardLinkParams): Promise<string | undefined> {
-  const environment =
-    process.env.APP_ENVIRONMENT === "production" ? "production" : "staging";
+  if (status === "online" && analyticsLink === null) {
```

Comment:
nit: Could we change this to a guard clause (inversing the condition) to save a level of indentation (and overhead) here?

```ts
// Before
if (condition) {
  doSomething()
} else { 
  return;
}

// After
if (!condition) return;

doSomething()
```